MedableResearchKit.xcodeproj/project.pbxproj

```diff
@@ -3535,16 +3532,11 @@
 				cs,
 				ca,
 				ar,
-				"zh-Hant",
-				"zh-HK",
-				"zh-Hans",
-				"pt-PT",
-				"fr-CA",
```

Comment:
Why fr-CA and pt-PT references were removed?

Reply:
They were duplicates as well.